ros_buildfarm/templates/status/release_status_page.html.em

```diff
@@ -60,6 +60,7 @@
       <a href="?q=ORANGE">ORANGE</a>,
       <a href="?q=YELLOW">YELLOW</a>,
       <a href="?q=GRAY">GRAY</a>
+      <a href="?q=UNMAINTAINED" title="Filter packages whth unmaintained or end-of-life status">UNMAINTAINED</a>
```

Comment:
Spelling: `whth` -> `with`

This query does not show packages with status "end of life". It only shows packages with status "unmaintained".